hrm-service/src/contact-job/contact-job-processor.ts

```diff
@@ -15,20 +17,16 @@ export function processContactJobs() {
     processingJobs = true;
 
     return setInterval(async () => {
-      // This is the primary job processing 'loop'
-      // First we process any completed jobs. These completed jobs would be pulled from a queue in prod but we just have a cheap in memory simulation using an array here
       try {
         // eslint-disable-next-line @typescript-eslint/no-unused-vars
-        const completedJobs = await processCompleteContactJobs();
+        const completedJobs = await pollAndprocessCompletedContactJobs();
```

Comment:
I don't think the "poll" step can be separated from the "complete" step, since SQS requires us to delete the processed messages manually.

Reply:
This will not be true anymore if we go with @robert-bo-davis's suggestion of exposing a REST endpoint to replace the polling of completed jobs.